### ISSUE TITLE
FileNotFound is just DokiDokiChomusukeClub?

### ISSUE THREAD

**Issue by Lil1th** [TheSwiftPhantom/ChomusukeTweaks#1]
![Screenshot (279)](https://user-images.githubusercontent.com/60187139/84406653-75bb2380-abcf-11ea-9fec-9abc7f2f6af2.png)

I just tried out FileNotFound but it seems to be identical to DDCC, even down to the Monika background. Do you mind if I post a fixed version like the below picture, or would you rather do it yourself? I can explain how it works if you'd like ^^

![Screenshot (277)](https://user-images.githubusercontent.com/60187139/84406684-7f448b80-abcf-11ea-82a6-24c8cc73aa6d.png)

**Comment by TheSwiftPhantom** [comment]
Basically :/ also please fix it (would take a screenshot but I can't find the file now... file not found :/ ) By the way, I have no idea how any type of code works. I almost fixed my Elizabeth III though :D (without the help of the internet :o )

**Comment by Lil1th** [comment]
Ok, so there's 5 things that I changed in the fixed version (which should show up on my page within 3m)

The first thing is I moved your 07.png into the extension-assets folder. index.css calls for background-image: url("../extension-assets/07.png"); which means that it is looking is extension-assets as opposed to FileNotFound's root folder.

Next, I just changed the names in tabby_bundle.js by replace-all'ing "Just" with "Error" and "Monika" with "404".

The third thing was changing the class attributes on .Lil-404 so that the shadow would be removed (or more accurately, changed to black which blends with the bg) and the cat would be made the blue of the Lil 404 cat. I also added   background-image: url("../extension-assets/07.png");
  background-size: 1600px;
  background-repeat: repeat; }
and removed the bracked after background-color: #A5C6EF; so it would recognize the image.

Fourth, I changed the pet-picker colour so that it would also display the Error 404 image:
.Lil-404 .alt-color--light, .Lil-404 .pet-picker__page:before...
    color: #000;
	background-image: url("../extension-assets/07.png");
  background-size: 1600px;
  background-repeat: repeat; }

And finally, I changed the font through Ferris so that it would be JetBrains Mono. Just for the more computer look.

DDCC is definitely the hardest to modify due to all the places I have messed around with the code in it (extension-assets, tabby_bundle.js, index.css, manifest.json, fonts, probably somewhere else too) but you're doing great on the other hacks, and I'm excited to see the final result for SiameseCat :)
